src/app/containers/AssignmentTemplateCreateForm.tsx

```diff
@@ -7,10 +7,35 @@ import { connect } from 'react-redux';
 import { RootState } from '../store';
 import { DaysOfWeek } from '../api/Api';
 import * as TimeUtils from '../infrastructure/TimeRangeUtils';
+import * as Validators from '../infrastructure/Validators';
 
 // wrapping generic assignment form in redux-form
 const formConfig: ConfigProps<any, AssignmentFormProps> = {
     form: 'CreateAssignmentTemplate',
+    validate: (values) => {
+        const errors: any = {};
```

Comment:
can we use the same validator used above?  It will require being defined in a static context.  What I would suggest is add it as a static method on the `AssignmentForm` similar to the parseValues functions